packages/react/src/cunningham-tokens.css

```
@@ -118,6 +118,8 @@
 	--c--components--forms-checkbox--border-color: #E7E8EA;
 	--c--components--forms-checkbox--border-radius: 2px;
 	--c--components--forms-checkbox--accent-color: #419A14;
+	--c--components--forms-checkbox--height: 1.5rem;
+	--c--components--forms-checkbox--width: 1.5rem;
```

I think we can assume we will always like a square.
```suggestion
	--c--components--forms-checkbox--size: 1.5rem;
```

---

Not false.